vizro-core/changelog.d/20240821_165515_petar_pejovic_return_empty_data_frame_from_tables_build.md

```diff
@@ -0,0 +1,47 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Changed
+
+- Return an empty data on initial build of AgGrid, Table and Figure objects. ([#644](https://github.com/mckinsey/vizro/pull/644))
+- Reduce number of figure function calls for AgGrid, Table and Figure objects. ([#644](https://github.com/mckinsey/vizro/pull/644))
```

Comment:
Let's try and make this a bit more user-facing and less implementation-specific. What is the actual impact of this change for users? Basically improved performance.

```suggestion
- Improve page loading time load for `AgGrid`, `Table` and `Figure`. ([#644](https://github.com/mckinsey/vizro/pull/644))
```